build

```
@@ -28,6 +28,27 @@ elif [ "$(uname -s)" = 'Linux' ]; then
     SYSTEM="$(echo "${DIST}${VERSION_ID}" | sed -r 's/([a-zA-Z]*)-.*/\1/g')"
 fi
 
+case "$(uname -m)" in
+    amd64)
+        ARCH=amd64
+        ;;
+    x86_64)
+        ARCH=amd64
+        ;;
+    arm64)
+        ARCH=arm64
+        ;;
+    aarch64)
+        ARCH=arm64
+        ;;
+    arm*)
+        ARCH=arm
+        ;;
+    *)
+        ARCH=$(uname -m)
+esac
+export ARCH
```

I think it's more clear if we use this 'rewrite' logic:
```suggestion
ARCH="$(uname -m)"
case "$ARCH" in
    x86_64)
        ARCH='amd64'
        ;;
    aarch64)
        ARCH='arm64'
        ;;
esac
export ARCH
```